lib/managers/files.js

```diff
@@ -1243,6 +1251,61 @@ Files.prototype.getCollaborations = function(fileID, options, callback) {
 };
 
 /**
+ * Enum of valid x-rep- hint values for generating representation info
+ *
+ * @readonly
+ * @enum {FileRepresentationType}
+ */
+Files.prototype.representation = {
+	PDF: '[pdf]',
+	THUMBNAIL: '[jpg?dimensions=320x320]',
+	IMAGE_MEDIUM: '[jpg?dimensions=1024x1024][png?dimensions=1024x1024]',
+	IMAGE_LARGE: '[jpg?dimensions=2048x2048][png?dimensions=2048x2048]',
+	EXTRACTED_TEXT: '[extracted_text]'
+};
+/**
+ * Requests information for all representation objects generated for a specific Box file
+ *
+ * API Endpoint: '/files/:fileID?fields=representations'
+ * Method : GET
+ *
+ * @param {string} fileID - Box ID of the file being requested
+ * @param {FileRepresentationType} representationType - The x-rep-hints value the application should create a
+ *    representation for. This value can either come from FileRepresentationType enum or manually created
+ * @param {Function} [callback] - A representation object containing the url_template, info, properties, and status
```

Comment:
This description is kind of weird; it's describing the value passed to the callback not the callback itself.  Also, the value passed to the callback isn't a single representation, it's a collection of representation objects